openmdao/code_review/test_lint_docstrings.py

```diff
@@ -543,12 +553,7 @@ def test_docstrings(self):
                         except:
                             continue
 
-                        for error_tuple in result['errors']:
-                            if error_tuple[0] not in ignore:
-                                if full_class_path not in failures:
-                                    failures[full_class_path] = []
-                                msg = f"{error_tuple[0]}: {error_tuple[1]}"
-                                failures[full_class_path].append(msg)
+                        failures.update(self._failure_dict(full_class_path, result))
```

Comment:
I think these `failures.update` calls should be inside the `try`...
if the `validate` call raises an exception, I'm guessing `result` is going to be `None`

(It looks like the only exception `validate` can raise is an [ImportError](https://github.com/numpy/numpydoc/blob/main/numpydoc/validate.py#L166) if the object can't be found)